package.json

```diff
@@ -128,7 +128,7 @@
     "https-proxy-agent": "5.0.1",
     "inquirer": "6.5.2",
     "inquirer-autocomplete-prompt": "1.4.0",
-    "ipx": "2.0.1",
+    "ipx": "2.1.0",
```

Comment:
Needed to have https://github.com/unjs/ipx/pull/189 and `allowAllDomains` option for IPX http storage actually working (otherwise it would need setting env var).

There is also new major of `ipx available overall, however I wanted to limit scope of this PR not to have to handle potential breaking changes